src/app/app.module.js

```diff
@@ -3,8 +3,18 @@ import uirouter from '@uirouter/angularjs';
 import ngMaterial from 'angular-material';
 import ngAnimate from 'angular-animate';
 import ngAria from 'angular-aria';
-import weatherService from './app.weatherService';
+import lastSearchesService from './services/lastSearches';
+import openWeatherMapsService from './services/openWeatherMaps';
+import startPage from './start-page/start-page';
+import lastSearches from './values/lastSearches';
+import currentWeather from './values/currentWeather';
+import "material-icons/iconfont/material-icons.scss";
+import "angular-material/angular-material.scss";
 
 export default angular
-    .module('app', [uirouter, ngMaterial, ngAnimate, ngAria])
-    .factory('weatherService', weatherService);
+    .module('startpage', [uirouter, ngMaterial, ngAnimate, ngAria])
```

Comment:
the 'uirouter, ngMaterial, ngAnimate, ngAria' will be used in all the project, I would maintain the 'app' module as a parent and import there this dependencies and your start page module